src/main/java/com/mycompany/myapp/service/reservation/ReservationService.java

```diff
@@ -24,14 +25,16 @@ public ReservationService(ReservationRepository reservationRepository, PoolRepos
         this.userService = userService;
     }
 
-    public Long reserve(String poolId, int count) throws Exception {
+    public Long reserve(String poolId, int count, Date from, Date to) throws Exception {
```

Comment:
Can we make it return `Reservation` object rather than id? It might be useful in the future

Reply:
kk

Reply:
kk